Units/parser-puppetManifest.r/puppet-definitionname.d/expected.tags

```diff
@@ -0,0 +1,11 @@
+t	input.pp	/^define t {$/;"	definition	line:2	language:PuppetManifest	end:4
+/tmp/multipledefinet	input.pp	/^    file { "\/tmp\/multipledefinet": content => "one" }$/;"	resource	line:3	language:PuppetManifest	scope:definition:t	end:3
+one_	input.pp	/^define one_ {$/;"	definition	line:6	language:PuppetManifest	end:8
+/tmp/multipledefineone_	input.pp	/^    file { "\/tmp\/multipledefineone_": content => "one" }$/;"	resource	line:7	language:PuppetManifest	scope:definition:one_	end:7
+on_e	input.pp	/^define on_e {$/;"	definition	line:10	language:PuppetManifest	end:12
+/tmp/multipledefineon_e	input.pp	/^    file { "\/tmp\/multipledefineon_e": content => "one" }$/;"	resource	line:11	language:PuppetManifest	scope:definition:on_e	end:11
+o_ne::tw_o	input.pp	/^define o_ne::tw_o {$/;"	definition	line:15	language:PuppetManifest	end:17
+/tmp/multipledefineo_netw_o	input.pp	/^    file { "\/tmp\/multipledefineo_netw_o": content => "two" }$/;"	resource	line:16	language:PuppetManifest	scope:definition:o_ne::tw_o	end:16
+o_ne::two_	input.pp	/^define o_ne::two_ {$/;"	definition	line:19	language:PuppetManifest	end:21
+/tmp/multipledefineo_netwo_	input.pp	/^    file { "\/tmp\/multipledefineo_netwo_": content => "two" }$/;"	resource	line:20	language:PuppetManifest	scope:definition:o_ne::two_	end:20
+t	input.pp	/^t { 't': }$/;"	resource	line:23	language:PuppetManifest	end:23
```

Comment:
This is probably another bug's indication. 
In the source file 5 different resources have been declared. 

```
t { 't': }
one_ { 'one_': }
on_e { 'on_e': }
o_ne::tw_o { 'o_ne::tw_o': }
o_ne::two_ { 'o_ne::two_': }
```
But this tag file only mentions one one them. 
I suspect there is another regular expression somewhere missing the "_" char. 

@masatake  if you point me to the right direction I can also fix that in another PR.